concent_api/core/tests/test_payments_database_transaction_storage.py

```diff
@@ -0,0 +1,193 @@
+import mock
+
+from django.test import TestCase
+from ethereum.transactions import Transaction
+from golem_messages.utils import decode_hex
+
+from core.models import GlobalTransactionState
+from core.models import PendingEthereumTransaction
+from core.payments.storage import DatabaseTransactionsStorage
+
+
+class DatabaseTransactionsStorageTest(TestCase):
+
+    multi_db = True
+
+    def setUp(self):
+        super().setUp()
+
+        with mock.patch('core.payments.service.get_transaction_count', return_value=5):
+            self.storage = DatabaseTransactionsStorage()
+
+        self.global_transaction_state = GlobalTransactionState.objects.get(pk=0)
+
+    def _create_pending_ethereum_transaction(self):
+        pending_ethereum_transaction = PendingEthereumTransaction(
+            nonce=self.global_transaction_state.nonce,
+            gasprice=10 ** 9,
+            startgas=21000,
```

Comment:
Please make at least one test that checks the full range of integers. We want to be sure that it does not break with very large numbers.

Reply:
Ok, but as you see in the discussion in other comments I am still not sure what are maximum values here.

Reply:
The sizes imply maximum values. For example for a 32-byte unsigned integer the maximum value is `2**(32*8)-1`